pytorch/pytorch
# Register operators of CV models in PyTorch mobile

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27554 [Lite Interpreter] Do not land, test models
* **#27379 Register operators of CV models in PyTorch mobile**

Currently the operators need to be registered manually through c10 registration. 

Tests:
The operators should be covered by tests on operators.
A few ops (add, conv) are covered in test_lite_interpreter.cpp for demonstration. 
CV models may be too large to include in unittests.
Simple local loaders can be built. Follow similar pattern as in test_lite_interpreter to
1. load the torch script model
2. run the model to get reference results
3. save and load the mobile module using torch::jit::module._save_for_mobile() and torch::jit::_load_for_mobile().
4. run the mobile module by run_method() and compare the results to reference results. 

Tested models:
* Lenet
* XrayMobileV3

Differential Revision: [D17810912](https://our.internmc.facebook.com/intern/diff/D17810912)
